software/featbit.yml

```
@@ -0,0 +1,11 @@
+name: Featbit
+website_url: https://www.featbit.co/
+description: Enterprise-grade feature flag platform that you can self-host. 
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - C#
```

```suggestion
  - K8S
```
I couldn't find how you can install it from source, but you can install it via a helm chart.

---

I went with C# as a requirement due to this. I agree lets remove it and replace it.